### DISCUSSION TITLE
django-tailwind-cli

### DISCUSSION THREAD

**Discussion by ryancheley** [django-commons/membership#124] [Check-ins]
Hi @django-commons/django-tailwind-cli! I wanted to take a moment and check in with you about how the transition to Django Commons has been.

- Have you run into any friction having your project in Django Commons, if so, can you describe them?
- Is there anything we can do to help your team or repo out?
- Do you have any questions or concerns for us?

If you'd like to chat with us privately, our email is [django-commons-admin@googlegroups.com](mailto:django-commons-admin@googlegroups.com). Thanks!

**Comment by oliverandrich** [comment]
Hey @ryancheley! Thanks for reaching out. The transition has been wonderful and went smoothly. So far I have only two topics on my task list for the project that related to django-commons.

- Find out how to link to the official code of conduct without having to copy it and still get the prominent link on the landing page of the project.
- Find a co-maintainer.

**Reply by cunla** [reply]
Re: link to code of conduct

You can have in your COC a link to the [COC in the membership repo](https://github.com/django-commons/membership/blob/main/CODE_OF_CONDUCT.md) - check out [playground-coc](https://github.com/django-commons/django-commons-playground/blob/main/CODE_OF_CONDUCT.md) as an example

**Reply by oliverandrich** [reply]
Thanks. This is a good advice and I just updated the CoC of the project accordingly.

**Comment by ryancheley** [comment]
@oliverandrich I’m so glad to hear that it’s going well! For the two points you’ve raised:

> * Find out how to link to the official code of conduct without having to copy it and still get the prominent link on the landing page of the project.

The `django-typer` package has a [GitHub Workflow](https://raw.githubusercontent.com/django-commons/django-typer/refs/heads/main/.github/workflows/update_coc.yml) that does this. The admins are also looking at ways to make this process easier not just for the CoC but other files as well. See [this issue](https://github.com/django-commons/controls/issues/65)

> * Find a co-maintainer.

The Admins are also talking about how to surface these types of things more easily within the organization. We don’t have any answers (yet), but it’s one of the things we’re hoping to be able to offer guidance on in the future.

**Comment by oliverandrich** [comment]
Just learned from your ticket, that there is a SECURITY.md file too. Can I just add it to my repository, or do I have to prepare the project for it?

**Reply by ryancheley** [reply]
As far as I know you can just add it to your repo, but I’ve only seen it on the `django-typer` project and not implemented it myself. There is a pretty good write up [here](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository) of the steps needed. Hope that helps!

**Comment by tim-schilling** [comment]
Hi @django-commons/django-tailwind-cli, we're circling back around :dizzy: We want to ask [Ryan's original questions](https://github.com/orgs/django-commons/discussions/124#discussion-7572872) to see if there's any update. We're also looking into how can we improve the new contributor experience with the overall goal of helping find regular contributor and more maintainers for projects.
- [Meet and greets](https://github.com/orgs/django-commons/discussions/136)
- [Project introduction videos](https://github.com/orgs/django-commons/discussions/150)

If any of that sounds interesting to you, feel free to engage on the discussions or here.

**Comment by williln** [comment]
Hi @django-commons/django-tailwind-cli -- this is our semi-annual checkin. If there is anything Django Commons can do to help you or your project, please let us know! Otherwise, happy coding ⭐

**Comment by oliverandrich** [comment]
Hi Lacey,

from my side, everything is fine and works like expected. Thanks for reaching out and the support.

Best regards,
Oliver

**Comment by tim-schilling** [comment]
Hi @django-commons/django-tailwind-cli this is just our semi-annual checkin. Let us know if there is anything Django Commons can do to help, or if there is anything else on your mind! Otherwise, happy coding 🥳

As an additional reminder, though not at all a suggestion/requirement. It may be worth looking through who has regularly contributed to your project to see if you'd like to elevate them to be a committer or a maintainer. I mention it because it's easy to forget to review the repositories community and helping people grow. Similarly, if you or anyone else wants to step back, that's fine too! If we can help coordinate, please let us know.

**Reply by tim-schilling** [reply]
Whoops, forgot some things. There are two other things:

1. Can the @django-commons/django-tailwind-cli-admins please share your emails with us? You can email us at [django-commons-admin@googlegroups.com](mailto:django-commons-admin@googlegroups.com) to avoid publishing publicly here.
2. We are working on a new policy around project abandonment. As a community organization, we need to have clear guidelines on when the community should pass the project onto someone else or signaling that the project is in a period of low activity. We'll share this new policy with you when we have it ready for comment. This will include a timeline for adoption. If you have questions, please let us know!
